utils/common.go

```diff
@@ -76,17 +77,18 @@ func MultipleChoiceInputReader(prompt string, choices []string) string {
 		fmt.Printf("You must provide an Integer. You provided %s", temp)
 		fmt.Println()
 		return MultipleChoiceInputReader(prompt, choices)
-	} else {
```

Comment:
@OleDakotaJoe 
Little refactored the code.
Also I think that there is a lack -- scanning input within `MultipleChoiceInputReader` that does not allow to create tests for the method.

What do you think about to factor `os.Stdin` out? And pass a reader as a method's parameter